package.json

```diff
@@ -1,17 +1,32 @@
 {
-  "name": "",
+  "name": "image_services",
   "version": "",
   "description": "",
-  "author": "",
+  "author": "Micko Cabacungan",
   "license": "",
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
+    "build": "webpack --watch -d",
+    "start": "node ./server/index.js",
+    "format": "prettier --write \"src/**/*.js\""
   },
-  "dependencies": {
-  },
+  "dependencies": {},
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "@babel/core": "^7.10.2",
+    "@babel/preset-env": "^7.10.2",
+    "@babel/preset-react": "^7.10.1",
+    "babel-loader": "^8.1.0",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
+    "express": "^4.17.1",
+    "html-webpack-plugin": "^4.3.0",
+    "mongoose": "^5.9.19",
+    "prettier": "2.0.5",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
react, react-dom, express, and mongoose should all be dependencies, not devDependencies

Reply:
Thank you! Will fix that :)